sample/src/main/res/values/styles.xml

```diff
@@ -3,6 +3,5 @@
         <item name="colorAccent">@color/color3</item>
         <item name="sonectColor3">@color/color3</item>
         <item name="sonectColor5">@color/color5</item>
-        <item name="sonectButtonTextColor">@color/white</item>
```

Comment:
<img width="482" alt="CleanShot 2021-03-18 at 15 45 10@2x" src="https://user-images.githubusercontent.com/1395176/111628147-f236fe80-8800-11eb-8d90-aea4643ee582.png">

This white is kind of correct. It affects everything and also this how partners will define buttonText color on other buttons.

For this control we'd use `sonectButtonTextColor` for text color for selected item and `sonectColor9` for non-selected item. We probably even could extract different style and/or color but as of now applying `sonectColor9` for non selected tab will be fine.

Reply:
Then I will apply this in the sdk-shop branch and create new PR.